test/cutensor.jl

```diff
@@ -450,6 +583,18 @@ end
             mC = reshape(permutedims(C, ipC), (loA, loB))
             @test mC ≈ mA * mB
 
+            # simple case with plan storage host-side
+            opA = CUTENSOR.CUTENSOR_OP_IDENTITY
```

Comment:
maybe we should `@enum_without_prefix cutensorOperator_t CUTENSOR_` so that you can do `CUTENSOR.OP_IDENTITY`

Reply:
Sounds good, but I think that can be a separate PR